src/layouts/home-assistant-main.html

```diff
@@ -81,6 +81,7 @@
 </dom-module>
 
 <script>
+var NON_SWIPABLE_PANELS = ['kiosk', 'map'];
```

Comment:
Don't add this as a global. Either put it as a variable on the Polymer object or wrap the whole Polymer declaration in an IIFE